package.json

```diff
@@ -74,14 +76,19 @@
     "express": "^4.17.1",
     "express-winston": "^4.1.0",
     "fs-extra": "^10.0.0",
+    "http-status-codes": "2.2.0",
+    "immutable": "^4.2.4",
     "js-yaml": "^4.1.0",
+    "kujira.js": "https://github.com/Team-Kujira/kujira.js",
```

Comment:
Importing contract addresses via a library like this `import contracts from 'kujira.js/src/resources/contracts.json';` is not secure, since it doesn't allow users to see what addresses they are interacting with and they can be modified without them seeing it. Remove this dependency and put the contract addresses as parameters in `kujira.yml` or in the Kujira connector code.

Reply:
This dependency actually cannot be removed. Kujira doesn't have an npm package and we need to import it's library from it's github.

Reply:
Regarding the `kujira.js/src/resources/contracts.json` we are actually taking care of that. By default we indeed import the contracts from the library, but the user also has options to use a local file or an external url. Also, for the tokens and the markets we have an important mechanism to allow the users to choose which tokens or markets they want to use, this avoids making the connector takes to long to initiate and loading too many markets or tokens; and also enables the user to use only the ones they specified. We can change the behavior to use, by default the contract from the url (we've done this way because for a default behavior its more reliable since an external url might change but library file will always be available).

Here's where the user can configure it:
```
tokens:
  url: # empty means using the default
#  url: /root/gateway/conf/connectors/kujira/tokens.json
#  url: https://raw.githubusercontent.com/Team-Kujira/kujira.js/master/src/resources/tokens.json
  allowed:
  disallowed:
  resolutionStrategy: markets
markets:
  url: # empty means using the default
#  url: /root/gateway/conf/connectors/kujira/contracts.json
#  url: https://raw.githubusercontent.com/Team-Kujira/kujira.js/master/src/resources/contracts.json
  allowed:
  disallowed:
```